hostsfile/hostname.sls

```diff
@@ -32,6 +32,8 @@ etc-sysconfig-network:
     - contents: {{ hostname }}
     - backup: false
 {% endif %}
+    - onchanges_in:
+      - cmd: set-fqdn
```

Comment:
this should be:
```
- cmd: hostsfile-set-fqdn
```

Reply:
Fixed inhttps://github.com/saltstack-formulas/hostsfile-formula/issues/39